android/BOINC/app/src/main/java/edu/berkeley/boinc/rpc/CcStateParser.java

```diff
@@ -132,11 +133,9 @@ public void startElement(String uri, String localName, String qName, Attributes
             mResultsParser.startElement(uri, localName, qName, attributes);
         }
         if (localName.equalsIgnoreCase("core_client_major_version") ||
-                localName.equalsIgnoreCase("core_client_minor_version") ||
-                localName.equalsIgnoreCase("core_client_release") ||
-                localName.equalsIgnoreCase("have_ati") ||
-                localName.equalsIgnoreCase("have_cuda")
-                ) {
+            localName.equalsIgnoreCase("core_client_minor_version") ||
```

Comment:
This is a type of change that makes things worse. The human editor had clear and justified intentions (with the line breaks) which should be preserved.